### ISSUE TITLE
Multiple errors if enabling circadian lights

### ISSUE THREAD

**Issue by mrDFX** [claytonjn/hass-circadian_lighting#161]
for example:
> 2021-06-07 21:05:14 ERROR (SyncWorker_28) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:16 ERROR (SyncWorker_29) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:17 ERROR (SyncWorker_6) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:18 ERROR (SyncWorker_4) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:19 ERROR (SyncWorker_0) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:21 ERROR (SyncWorker_32) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:22 ERROR (SyncWorker_22) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:23 ERROR (SyncWorker_23) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:25 ERROR (SyncWorker_13) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:26 ERROR (SyncWorker_3) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:27 ERROR (SyncWorker_17) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -5006, 'message': 'illegal request'}
> 2021-06-07 21:05:28 ERROR (SyncWorker_12) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_brightness at 0x7f68436228b0>: {'code': -1, 'message': 'client quota exceeded'}
> 2021-06-07 21:05:29 ERROR (SyncWorker_12) [homeassistant.components.yeelight.light] Error when calling <function YeelightGenericLight.set_default at 0x7f6843622c10>: {'code': -1, 'message': 'client quota exceeded'}

**Comment by claytonjn** [comment]
Why are you using such an old version of Circadian Lighting? v1 does not work with Home Assistant 2021.5.x+, so it's almost definitely not caused by CL as v1 of the component doesn't even load on that version of Home Assistant.

**Comment by mrDFX** [comment]
Troubles started after i'm upgrade HA, and plugin Circadian Lighting(then still on version 1.0.11) cant use my yeelight lamps - lamps go to unavailable state
after this upgrade HA - next i'm try upgrade Circadian Lighting to 1.0.13b - and Circadian Lighting plugin return same effects..
After released 2.0.4 i'm upgrade Circadian Lighting - but this not solve my troubles..
now i'm cant use Circadian Lighting plugin :(

**Comment by mrDFX** [comment]
@claytonjn, Maybe i'm can provide more logs or other details for resolve this issue?

**Comment by claytonjn** [comment]
Yes, it would be helpful if you turned on [debug logging](https://github.com/claytonjn/hass-circadian_lighting/wiki/Debugging) for Circadian Lighting and could post a log that includes both the CL debug log entries and the errors you're seeing from your lights.
